cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter uses the Roles and Organization Management outlined in [sig-governance](https://github.com/spinnaker/governance/blob/master/governance.md).
+
+## Scope
+
+The Contributor Experience Special Interest Group (SIG) is responsible for improving the experience of those who contribute upstream to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues](https://github.com/spinnaker/spinnaker/issues), we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Projects & Services
+
+-  In accordance with the code of conduct, establish policies, standards, and procedures for the use, moderation, and management of all public platforms officially used by the project, including but not limited to:
```

Comment:
I think some examples of the types of policies that might be implemented would be useful.  Depending on the particular policy, I would imagine that we might want to have this SIG work on building consensus and making a proposal, then presenting that to the SC/TOC for comments/final approval.

I don't want to add extra steps for small tactical procedures, but do want to make sure that we are appropriately reviewing changes that have a broader impact (and thus why understanding some of the types of policies would be useful).

Reply:
Nikema is working on adding some more specifics to the doc in response to your suggestion. One thing I'll say generally is, the model we discussed with Rob in his comments is more of an "opt-in" model than an approval gate model. This SIG can establish tools/resources/processes to enable SIGs to manage things or make them more welcoming/equitable. Then next steps would be to pitch or offer those things to the SIGs themselves. If SIGs choose to adopt them, great. Successful SIGs can be a model to non-adopters if the strategies are helpful. That way buy-in happens over time.

I could see an instance in which this SIG might discover something that really should be addressed project-wide and adopted by all SIGs. In those cases, presenting a proposal or RFC to the TOC/SC for approval seems reasonable.

Thoughts @ezimanyi ?

Reply:
This model sounds great to me!  I really like the idea of this SIG helping other SIGs with best practices and tooling and agree (at least initially) that this would be best as opt-in. Just read through the text changes to the RFC which look good.